src/metabase/models/channel.clj

```diff
@@ -44,3 +45,7 @@
   [instance]
   (assert-channel-type instance)
   instance)
+
+(defmethod audit-log/model-details :model/Channel
+  [channel _event-type]
+  (select-keys channel [:id :name :description :type :active]))
```

Comment:
How do you decide what data should go in the details? I'm not sure how to review this list.

Compared to other models like `:model/Card` and `:model/Table` it seems like `:active` is unusual to include in the details. But I'm not sure

Reply:
yea, IMO even detail should be included.

Asked about it [here](https://metaboat.slack.com/archives/C01LQQ2UW03/p1721878298744049)

But fwiw: @luizarakaki  is 👍 with the list https://metaboat.slack.com/archives/C07318NQ5M2/p1721878597825819

Reply:
> Any user update should add audit logs

that is surprising given how slim the other `model-details` implementations are.